Makefile

```diff
@@ -0,0 +1,72 @@
+.DEFAULT_GOAL = help
+.PHONY: help install gitmodules build start stop wait-healthy sh exec logs watch run dev prod
+
+SHELL = /usr/bin/env bash
+
+ifneq (${TARGET}, prod)
+TARGET = dev
+endif
+
+DOCKER_COMPOSE = docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.${TARGET}.yml
+
+STOP = exit=$$?; $(MAKE) stop; exit $$exit
```

Comment:
Is this a trap to bring down the containers if something fails? If so: TIL something awesome.

Reply:
Yep! Even a Ctrl + C won't stop them normally.

Reply:
Wicked.